irina/ex.js

```diff
@@ -0,0 +1,98 @@
+//Nests
+
+/*
+https://www.javascripttutorial.net/javascript-multidimensional-array/
+The dimension of an array indicates the number of indices you need to select an element.
+For a two-dimensional array you need two indices to select an element
+For a three-dimensional array you need three indices to select an element
+ */
+
+// 1. 2D Array
+// Given the 2D Array below, loop through the arrays to print the values.
+const board = [
```

Comment:
The first method is looping through the arrays and prints the values but you are also creating e new array and pushing all the values into the new one. Is not wrong but you are doing more work the second exercises is also correct and to the points of the exercises.